app/models/edm/web_resource.rb

```diff
@@ -63,9 +79,9 @@ def edm_type_from_media_content_type
 
     ##
     # Validation method for the web resource's media to only allow certain types of content.
-    # TODO: Make these restrictions more specific, potentially move to campaign specific validations.
     def europeana_supported_media_mime_type
-      errors.add(:media, I18n.t('errors.messages.inclusion')) unless media&.content_type&.match?(%r{\Aimage/|\Aaudio/})
+      content_regex = %r{#{'\A' + ALLOWED_CONTENT_TYPES.join('\z|\A') + '\z'}}
+      errors.add(:media, I18n.t('errors.messages.inclusion')) unless media&.content_type&.match?(content_regex)
```

Comment:
No need for a regex here. Just use `ALLOWED_CONTENT_TYPES.include?(media&.content_type)`